tests/unit/test_alpha.py

```diff
@@ -10,17 +10,23 @@
 from __future__ import absolute_import, print_function, unicode_literals
 
 import pytest
+import os
 
 from hepcrawl.spiders import alpha_spider
 
-from .responses import fake_response_from_file
+from hepcrawl.testlib.fixtures import fake_response_from_file
 
 
 @pytest.fixture
 def results():
     """Return results generator from the Alpha spider."""
     spider = alpha_spider.AlphaSpider()
-    records = list(spider.parse(fake_response_from_file('alpha/test_1.htm')))
+    records = list(
+        spider.parse(
+            fake_response_from_file('./../../tests/unit/responses/alpha/test_1.htm')
```

Comment:
Did you forget to add the new module file on the testlib module?

Also, we should handle the path resolution on the function, so here we can just pass a simpler path, like:
```python
fake_response_from_file('alpha/test_1.htm')
```